Editor/GameLiftClientSettingsEditor.cs

```diff
@@ -10,33 +10,29 @@ namespace AmazonGameLift.Editor
     [CustomEditor(typeof(GameLiftClientSettings))]
     public sealed class GameLiftClientSettingsEditor : UnityEditor.Editor
     {
-        private SerializedProperty _isLocalTest;
```

Comment:
Is there a use case where someone may want to run the unity plugin with sdk4 or are we completely deprecating support?

Reply:
From what @xmiao2 has said, this wouldn't be backwards compatible.